src/test/groovy/com/causecode/logger/ReplaceSlf4jLogger.groovy

```diff
@@ -0,0 +1,40 @@
+package com.causecode.logger
```

Comment:
CR

Reply:
We are not adding CR in any open source projects.